master/static/srv/get_experiments.sql

```diff
@@ -1,6 +1,8 @@
 WITH filtered_exps AS (
     SELECT
         e.id AS id,
+        COALESCE(NULLIF(e.config->>'name', ''), NULLIF(e.config->>'description', ''), 'Experiment ' ||  e.id) AS name,
+        e.config->>'notes' AS notes,
```

Comment:
Thinking about it now, should notes be in the config? Should notes be transferred on a fork, or should they start over. It feels like maybe they should start over but we should ask an AMLE. wdyt?

Reply:
the idea was that they'd have the option of setting it in their experiment config. although since we want to support annotations for notes that might be a bit tricky to manually embed in a yaml or json config file.